docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
I found that somewhat confusing when I read it the first time; I interpreted it as referring to a package `Example.jl` which we're creating, which could (in principle) be living anywhere and not necessarily in one of the locations at LOAD_PATH. Especially because the example mentions using Documenter.jl with an arbitrary (single) Julia script.

I now understand better the perspective from which the document was written, and I wonder if it would be helpful to have a clarification near the statement you pointed to, because I completely missed the significance of what is assumed there.